Code/Framework/AzManipulatorTestFramework/Include/AzManipulatorTestFramework/ActionDispatcher.h

```diff
@@ -92,8 +92,7 @@ namespace AzManipulatorTestFramework
     {
         if (m_logging)
         {
-            AZStd::string message = AZStd::string::format(format, args...);
-            AZ_Printf("[ActionDispatcher] %s", message.c_str());
```

Comment:
I am not sure what the intention was here. The first parameter is the "window" which is just a categorization, the second parameter is the format.